aws-android-sdk-core/src/main/java/com/amazonaws/Request.java

```diff
@@ -61,13 +61,15 @@
      *
      * @param path The path to the resource being requested.
      */
+    @Deprecated
```

Comment:
I don't think we should mark this as deprecated, since we still plan to rely on this method for the majority of the existing SDKs.

Reply:
This method shouldn't be used for newer implementation though. Otherwise we will be struggling with encoding issues.

Reply:
As discussed, it's really just a question of whether we want a warning to show up.  Thinking about it some more, I do think it's a good idea to mark the old ones as deprecated, so that when codegen adds support for new SDKs, they opt for the new approach.